packages/bsky/src/hydration/hydrator.ts

```diff
@@ -409,6 +412,8 @@ export class Hydrator {
       this.hydrateFeedGens(nestedFeedGenUris, ctx),
       this.hydrateLabelers(nestedLabelerDids, ctx),
       this.hydrateStarterPacksBasic(nestedStarterPackUris, ctx),
+      // only gates for quoted posts
+      this.feed.getPostGatesForPosts(postUrisLayer1),
```

Comment:
this var could use a better name, had to really look through the code to figure out what it was. it only includes quoted posts right? maybe something like `embedPostUrisLayer1`?